_posts/2020-10-29-Rails-6-1-beta1-release.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: 'Rails 6.1: Horizontal Sharding, Multi-DB Improvements, Strict Loading, Destroy Associations in Background, Error Objects, and more!'
+categories: releases
+author: eileencodes
+published: true
+date: 2020-10-29 10:00:00 -05:00
+---
+The first beta for Rails 6.1 has been released and wow does it have a lot of great stuff! We've been hard at work these past few months implementing improvements to multiple databases, adding support for destroying associations in jobs instead of in memory, turning errors into objects, and so much more.
```

Comment:
```suggestion
The first beta for Rails 6.1 has been released and wow does it have a lot of great stuff! We've been hard at work these past few months implementing improvements to multiple databases, adding support for destroying associations in jobs instead of in-memory, turning errors into objects, and so much more.
```